src/components/Dropdown/Dropdown.tsx

```diff
@@ -32,60 +41,84 @@ export const Dropdown = ({
     onChange,
     activeItemId = "",
     placeholder = "Select item",
-    size = Size.Small,
+    size = DropdownSize.Small,
     disabled = false,
     clearable = false,
+    ariaLabel = "Dropdown",
 }: DropdownProps): ReactElement<DropdownProps> => {
-    const [isOpen, setIsOpen] = useState(false);
     const dropdownElement = useRef<HTMLDivElement | null>(null);
-    useClickOutside(dropdownElement.current, () => setIsOpen(false));
-
     const activeItem = getActiveItem(menuBlocks, activeItemId);
+    const props = mapToAriaProps(ariaLabel, menuBlocks);
+    const state = useSelectState({
+        ...props,
+        defaultSelectedKey: activeItemId,
+        onSelectionChange: (key) => onChange(`${key}`),
```

Comment:
no need to use template literal

Reply:
The key could also be a number so there's a type error without the template literals.